ee/tasks/usage_report.py

```diff
@@ -430,3 +355,65 @@ def fetch_sql(sql_: str, params: Tuple[Any, ...]) -> List[Any]:
     with connection.cursor() as cursor:
         cursor.execute(sql.SQL(sql_), params)
         return namedtuplefetchall(cursor)
+
+
+def send_org_usage_report(
+    organization_id: Optional[str] = None, dry_run: bool = False, at: Optional[str] = None
+) -> Dict:
+
+    at_date = dateutil.parser.parse(at) if at else None
+    period = get_previous_day(at=at_date)
+    period_start, _ = period
+
+    license = License.objects.first_valid()
+    metadata = get_instance_metadata(period, bool(license))
+
+    organization = Organization.objects.get(id=organization_id)
+    organization_id = str(organization.id)
+    teams = organization.teams.all()
+    team_ids = [team.id for team in teams]
+
+    try:
+        org_owner = get_org_owner_or_first_user(organization_id)
+
+        if not org_owner:
+            raise Exception("No owner found for organization")
+
+        logger.info("Sending usage report for organization %s ...", organization_id)
+
+        distinct_id = org_owner.distinct_id
+        usage = get_org_usage_report(period, team_ids)
```

Comment:
We should have a minimum set of usage data (events, recordings counts) and just send these if anything else is failing

Reply:
Agreed but going to do that as a separate PR as it needs some thinking...